ios/OX Coi/Application/AppDelegate.swift

```diff
@@ -34,8 +36,35 @@ class AppDelegate: FlutterAppDelegate {
                 } else {
                     result(nil)
                 }
+            } else if call.method.contains(Method.Sharing.SendSharedData) {
+                self.shareFile(arguments: call.arguments as! Dictionary<String, String>)
+                result(nil)
             }
         }
     }
+    
+    private func shareFile(arguments: Dictionary<String, String>) {
+        let args: [String: String] = arguments
```

Comment:
...get rid of this superfluous declaration. 😊

Reply:
Done